persistent-mysql/ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+## Master
```

Comment:
This is fine, but typically we just go ahead and bump the version number based on the kind of change. In this case, it would be a minor version bump (e.g. 2.1.2.1 ==> 2.1.3).